web/src/components/profile/getErrorStateForAddressField.ts

```diff
@@ -78,18 +78,28 @@ export const getErrorStateForAddressField = ({
 
   if (field === "addressLine1") {
     const maxLengthError = fieldWithMaxLength({ required: false, maxLen: BUSINESS_ADDRESS_LINE_1_MAX_CHAR });
-
-    const partialAddressError = fieldWithAssociatedFields({
-      associatedFields: [
-        "addressMunicipality",
-        "addressZipCode",
-        "addressLine2",
-        "addressCity",
-        "addressProvince",
-        "addressCountry",
-      ],
-      label: (Config.formation.fields as any)[field].error,
-    });
+    let partialAddressError;
```

Comment:
There is an issue here, if a user forms their business, then visits the tax clearance anytime action, then the error handling is handled differently than a person who forms then visit the tax clearance anytime action.